FAQ.md

```diff
@@ -790,6 +908,20 @@ The monitoring system of TiDB consists of Prometheus and Grafana. From the dashb
 
 Yes. Find the startup script on the machine where Prometheus is started, edit the startup parameter and restart Prometheus.
 
+#### Region Health monitor
+
+In TiDB 2.0, Region health is monitored in the PD metric monitor page, in which the `Region Health` monitor item shows the statistics of all the Region replica status. `miss` means shortage of replicas and `extra` means the extra replica exists. In addition, `Region Health` also shows the isolation level by `label`. `level-1` means the Region replicas are isolated physically in the first `label` level. All the Regions are in `level-0` when `location label` is not configured.
+
+#### What is the meaning of `selectsimplefull` in Statement Count monitor?
+
+It means full table scan but the table may be a small system table.
+
+#### What is the difference between `QPS` and `Statement OPS` in the monitor?
+
+The `QPS` statisctics is about all the SQL statements, including `use database`, `load data`, `begin`, `commit`, `set`, `show`, `insert` and `select`.
+
+The `Statement OPS` statistics is only about service related SQL statements, including `select`, `update` and `insert`, therefore the `Statement OPS` statistics matches the service.
```

Comment:
- service related -> applications related
- matches the service -> matches the applications better